app/helpers/hyrax/hyrax_helper_behavior.rb

```diff
@@ -233,6 +250,16 @@ def user_agent
         request.user_agent || ''
       end
 
+      def count_classes_for(unread_count)
+        'count label '.tap do |classes|
```

Comment:
Always use the `count` and `label` css classes. Otherwise, vary the classes depending on the unread count.